utilities/PerformanceTesting/start-secured-cluster.sh

```diff
@@ -9,7 +9,6 @@ else
     collector_image_registry=${2:-quay.io/rhacs-eng/$ARCH}
 fi
 
-
```

Comment:
cosmetic

Reply:
Already fixed. Cant see this in the "Files Changed" tab.